.gitignore

```diff
@@ -48,6 +48,16 @@ TestSimpleMain
 # 3rd Party builds
 build_ext
 
+# Visual Studio
+.vs
+Debug
+**/.vscode
+*.vcxproj*
+*.sln
+*.exp
+*.pdb
+*.sln
+
```

Comment:
Don't do that. Just mkdir build somewhere, cmake -G MSVC <sourcedir>, and play with compilation here.

Reply:
I'm okay with eventually having these in the gitignore, I can see us having a `generate-solution.bat` file eventually that runs the CMake command to make these files for people